docs/07-deployments/04-deploying-to-aws.md

```diff
@@ -118,6 +132,27 @@ _Connect to the database with Postico._
 
 ## Deploy your code
 
+:::caution
+
+using an old version of serverpod cli will generate the github action file containing old dart version. You can make the following fixes:
+
+1. in `./github/workflows/deployment-aws.yml` use
+
+```yaml
+      - name: Setup Dart SDK
+        uses: dart-lang/setup-dart@v1.6.5
+        with:
+          sdk: 3.5
```

Comment:
Suggetsion: Add a comment here saying that you should use the dart version you want to run your project in

e.g.
```
# Set your desired dart version here
```